DemoBoolean.java

```diff
@@ -0,0 +1,11 @@
+package com.edocti.jintro.lab01;
+	
+public class DemoBoolean{
+	public static void main(String[] args){
+		boolean isOld=(age > 65);
```

Comment:
paranthesis is ok, but not necessary

boolean isOld = age > 65;

Also, notice the spaces